Description: Version path
 Fix the path to the version file
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2014-12-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/redis/rack/version_test.rb
+++ b/test/redis/rack/version_test.rb
@@ -1,5 +1,5 @@
 require 'test_helper'
-
+require 'redis/rack/version'
 describe Redis::Rack::VERSION do
   it 'returns current version' do
     Redis::Rack::VERSION.must_equal '1.5.0'
